Song_sort_Genre.php

```diff
@@ -42,7 +73,10 @@
 			<h3>Genre</h3>
 			<h3>Duration</h3>
 			<h3>Size</h3>				
+		</div>
+		<div class="content_4">
```

Comment:
21/9: I break the content_3 div box to content_3 and content_4. For content_3 div box I decided for it to hold the title and content_4 div box hold the data that is been import from the database